lib/parser/response.rb

```diff
@@ -1,21 +1,49 @@
+require 'cgi'
+require 'beta_code'
+
 require_relative './word'
 require_relative './unknown'
 
 module Parser
+  CREATED_DATE = '2020-01-01T00:00:00.000000'
+
   class Response
-    def initialize(doc)
+    def initialize(word, latin, doc)
+      @word = CGI.escapeHTML(word)
+      @latin = latin
+
       @words = doc.xpath('./words/word').map { |w| Word.new(w) }
       @unknowns = doc.xpath('./words/unknown').map { |u| Unknown.new(u) }
     end
 
-    def to_xml
-      @to_xml ||= "<words>\n#{components.map(&:to_xml).join}</words>"
+    def bamboo_xml
+      @bamboo_xml ||= <<~XML
```

Comment:
One weakness of this code (since it uses strings instead of building an XML object) is that it assumes the output of Morpheus is valid XML. If Morpheus doesn't escape something correctly that error would not be caught.